packages/cxl-ui/src/components/partials/cxl-single-multiversion-card-display.js

```diff
@@ -0,0 +1,106 @@
+import {html} from 'lit-html';
+import {unsafeHTML} from 'lit-html/directives/unsafe-html';
+
+const CXLSingleMultiversionCardDisplay = (el, userId, selected) => {
+  const postType = el.cxl_hybrid_attr_post['@attributes'].class.includes(
+    'category-minidegree-programs'
+  )
+    ? 'minidegree'
+    : 'playbook';
+
+  return html`
+    <cxl-accordion-multiversion-card
+      id="${el.cxl_hybrid_attr_post['@attributes'].id}"
+      class="${el.cxl_hybrid_attr_post['@attributes'].class}"
+      theme="${postType === 'minidegree' ? 'dark' : ''}"
+    >
+      <header class="entry-header" slot="summary">
+        <label title="${postType === 'minidegree' ? 'Course' : 'Playbook'}"
+               class="entry-type">${postType === 'minidegree' ? 'COURSE' : 'PLAYBOOK'}</label>
+
+        <cxl-save-favorite
+          postType="${postType === 'minidegree' ? 'minidegree' : 'playbook'}"
+          postId="${el.cxl_hybrid_attr_post['@attributes'].id}"
+          userId="${userId}"
+          ?selected=${selected}
+          isCardVersion
+        ></cxl-save-favorite>
+
+        <div class="entry-summary" itemprop="description">${el.breadcrumbs.join('/')}</div>
+
+        <h2 class="entry-title no-anchor" itemprop="headline">
+          <a href="${el.conversionxl_certificate_sales_page}" rel="bookmark" itemprop="url"
```

Comment:
`conversionxl_certificate_sales_page` is very specific name and it's a bit confusing because `certificate`, `minidegree` and `playbook` are used here. `conversionxl_certificate_sales_page` indicates it's certificate specific, but as far as I understand it's just URL to the post of any type.